dags/hivemind_etl_helpers/tests/integration/test_github_etl_fetch_raw_issues.py

```diff
@@ -60,7 +63,7 @@ def test_get_single_issue_single_repo(self):
             )
 
         repository_ids = [123]
-        issues = fetch_raw_issues(
+        issues = self.extractor._GithubIssueExtraction__fetch_raw_issues(
```

Comment:
Please consider updating to use the `_fetch_raw_issues` directly and not via `GitHubIssueExtraction`